qemu/tests/qemu_guest_agent.py

```diff
@@ -3288,13 +3288,13 @@ def gagent_check_user_logoff(self, test, params, env):
         login_user_id = user_info.strip().split()[2]
 
         error_context.context("Make the user log out.", logging.info)
-        try:
+        #try:
             session.cmd("logoff %s" % login_user_id)
-        except aexpect.ShellProcessTerminatedError as detail:
-            if not re.search("Connection reset by peer", str(detail)):
-                test.error("Error occured with %s." % str(detail))
-        else:
-            test.fail("The user logoff failed.")
+        #except aexpect.ShellProcessTerminatedError as detail:
+        #    if not re.search("Connection reset by peer", str(detail)):
+        #        test.error("Error occured with %s." % str(detail))
```

Comment:
Only need to change here.
```suggestion
        except (aexpect.ShellTimeoutError,
                aexpect.ShellProcessTerminatedError,
                aexpect.ShellStatusError):
            test.error("Error occured with %s." % str(detail))
```